arch/xtensa/src/esp32/esp32_rt_timer.c

```diff
@@ -281,8 +282,8 @@ static void delete_rt_timer(FAR struct rt_timer_s *timer)
  * Name: rt_timer_thread
  *
  * Description:
- *   RT timer working thread, it wait for a timeout semaphore, scan
- *   the timeout list and process all timers in this list.
+ *   RT timer working thread, it waits for a timeout semaphore, scans
```

Comment:
```suggestion
 *   RT timer working thread: Waits for a timeout semaphore, scans
```
Copying the modification from the other file.